pages/knowledge/web-components/README.md

```diff
@@ -0,0 +1,49 @@
+---
+navigation_title: 'Web Components'
+position: 5
+---
+
+# Web Components
+
+**"Web Components" is an umbrella term for three main technologies: Custom elements, shadow DOM and HTML templates. Using these technologies allows us to create reusable, optionally encapsulated components. There are some important caveats regarding accessibility we need to keep in mind when authoring web components.**
+
+[[_TOC_]]
+
+## The main technologies
+
+[mdn](https://developer.mozilla.org/en-US/docs/Web/API/Web_components) has the most useful overview of the underlying technologies. It is important to understand that they can be used independently of each other. There is absolutely no need to use shadow DOM when writing web components. At the same time, shadow DOM can be used outside of custom elements, too.
```

Comment:
```suggestion
[MDN](https://developer.mozilla.org/en-US/docs/Web/API/Web_components) has the most useful overview of the underlying technologies. It is important to understand that they can be used independently of each other. There is absolutely no need to use shadow DOM when writing web components. At the same time, shadow DOM can be used outside of custom elements, too.
```
Very, very picky: MDN is usually written in capital letters (even if it doesn't apply to the logo  🙈).
See e.g. https://developer.mozilla.org/en-US/about

(Of course, lines 20, 26 and 33 would also have to be adjusted accordingly.)

Reply:
Oh, you're absolutely right, fixed.